staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5215,19 +5215,8 @@ type EventSeries struct {
 	Count int32 `json:"count,omitempty" protobuf:"varint,1,name=count"`
 	// Time of the last occurrence observed
 	LastObservedTime metav1.MicroTime `json:"lastObservedTime,omitempty" protobuf:"bytes,2,name=lastObservedTime"`
-	// State of this Series: Ongoing or Finished
-	// Deprecated. Planned removal for 1.18
-	State EventSeriesState `json:"state,omitempty" protobuf:"bytes,3,name=state"`
```

Comment:
tombstone the field so we don't reuse this proto or json tag. examples:

https://github.com/kubernetes/kubernetes/blob/e97c570a4ba5ba1e2285d3278396937feaa15385/staging/src/k8s.io/api/core/v1/types.go#L4311-L4313

https://github.com/kubernetes/kubernetes/blob/e97c570a4ba5ba1e2285d3278396937feaa15385/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go#L1118-L1119

Reply:
Thanks for catching.